medinify/datasets/review_dataset.py

```diff
@@ -75,6 +75,46 @@ def collect_all_common_reviews(self, start=0):
 
         print('\nAll common drug review scraping complete!')
 
+    def collect_urls(self, file_path, start=0):
+        """Scrape all reviews for all drugs urls in file
+
+        Args:
+            start: index to start at if continuing from previous run
+        """
```

Comment:
Have you tested this with picking up at a certain index? I think you'll need to use load to not overwrite older reviews.

Reply:
I think that these changes will fix that issue. I tested stopping and restarting then loading the file and it looks like it contains all the reviews.